### PR TITLE
chore: inject lens icon on twitter

### PR DESCRIPTION
**Pull request by UncleBill**

### CODE REVIEW

File: packages/shared/src/UI/components/SocialAccountList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                        className={classes.socialAccountListItem}
+                        disabled={false}
+                        onClick={() => openWindow(resolveNextIDPlatformLink(x.platform, x.identity, x.name))}>
+                        {Icon ? <Icon size={20} /> : null}
+                        <Typography className={cx(classes.socialName, classes.accountNameInList)}>
+                            {x.name || x.identity}
+                        </Typography>
+                        {x.platform === NextIDPlatform.LENS ? (
+                            <Button variant="text" className={classes.followButton} disableElevation>
+                                Follow
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/SocialAccountList/index.tsx:141]: i18n

File: packages/shared/src/UI/components/SocialAccountList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                border: '7px solid rgba(0, 0, 0, 0)',
+                backgroundColor: theme.palette.maskColor.secondaryLine,
+                backgroundClip: 'padding-box',
+            },
+        },
+        followButton: {
+            marginLeft: 'auto',
+            height: 32,
+            padding: theme.spacing(1, 2),
+            backgroundColor: '#ABFE2C',
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/SocialAccountList/index.tsx:92]: Avoid dead-coded hex colors.

File: packages/shared/src/UI/components/SocialAccountList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+            scrollbarColor: `${theme.palette.maskColor.secondaryLine} ${theme.palette.maskColor.secondaryLine}`,
+            scrollbarWidth: 'thin',
+            '::-webkit-scrollbar': {
+                backgroundColor: 'transparent',
+                width: 19,
+            },
+            '::-webkit-scrollbar-thumb': {
+                borderRadius: '20px',
+                width: 4,
+                border: '7px solid rgba(0, 0, 0, 0)',
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/SocialAccountList/index.tsx:83]:
```suggestion
                border: '7px solid transparent',
```

File: packages/shared/src/UI/components/SocialAccountList/SocialTooltip.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+}: PropsWithChildren<SocialTooltipProps>) {
+    const { classes } = useStyles({ hidden })
+    const t = useSharedI18N()
+    return (
+        <ShadowRootTooltip
+            classes={{ tooltip: classes.tooltip, arrow: classes.arrow }}
+            arrow
+            placement="top"
+            title={
+                <Typography style={{ padding: '6px 2px', whiteSpace: 'nowrap' }} fontSize={14}>
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/SocialAccountList/SocialTooltip.tsx:42]: The default font-size is `14px`.
```suggestion
                <Typography style={{ padding: '6px 2px', whiteSpace: 'nowrap' }}>
```

File: packages/plugins/Handle/src/SNSAdaptor/SocialAccountList.tsx [hunk 5]
```diff
@@ -74,17 +77,26 @@ const useStyles = makeStyles<StyleProps>()((theme, { isMenuScroll = false }) =>
                 backgroundClip: 'padding-box',
             },
         },
+        followButton: {
+            marginLeft: 'auto',
+            height: 32,
+            padding: theme.spacing(1, 2),
+            backgroundColor: '#ABFE2C',
```
Comment (guanbinrui) [inline packages/plugins/Handle/src/SNSAdaptor/SocialAccountList.tsx:84]: Avoid using dead-coded hex colors.

File: packages/plugin-infra/src/types.ts [hunk 6]
```diff
@@ -484,6 +484,8 @@ export namespace Plugin.SNSAdaptor {
         // }
         /** This UI will be rendered components on the tips realm */
         TipsRealm?: TipsRealm
+        /** This UI will be rendered components on the tips realm */
```
Comment (guanbinrui) [inline packages/plugin-infra/src/types.ts:475]: Align this comment.

**Comment by Jack-Works** [comment]
do you sure this pr has the correct base branch? usually, you should not target the `released` branch. it's only for a new version. also, this pr contains too many conflicts

**Pull request closed without merging**